### PR TITLE
badge unit testing

### PR DESCRIPTION
**Pull request by brianchristopherbrady**
adds Badge web component playwright tests

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b03b40647507909a151f966be6cd67b65cfcf3f9'>b03b40647507909a151f966be6cd67b65cfcf3f9</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/badge/badge.spec.ts [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    });
+    await expect(element).toHaveAttribute('color', 'subtle');
+    await expect(element).toHaveJSProperty('color', 'subtle');
+
+    await element.evaluate((node: Badge) => {
+      node.color = 'subtle';
+    });
+    await expect(element).toHaveAttribute('color', 'subtle');
+    await expect(element).toHaveJSProperty('color', 'subtle');
+  });
```
Comment (procload) [inline packages/web-components/src/badge/badge.spec.ts:87]: These are duped. I think we're missing warning.

File: packages/web-components/src/badge/badge.spec.ts [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    element = page.locator('fluent-badge');
+    root = page.locator('#root');
+
+    await page.goto(fixtureURL('components-badge--badge'));
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
```
Comment (procload) [inline packages/web-components/src/badge/badge.spec.ts:23]: We might consider adding a test for the default content to make sure it renders properly in the slot.

**Review by procload — CHANGES_REQUESTED** [review]

**Review by halerankin — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by miroslavstastny — APPROVED** [review]

**Review by procload — APPROVED** [review]

**Pull request merged**